src/utils/py/daos_api.py

```diff
@@ -799,7 +725,7 @@ def create(self, rank=None, objcls=None):
 
         # Convert the object class into an valid object class enumeration value
         if objcls is None:
-            obj_cls_int = DaosObjClass.OC_RP_XSF.value
+            obj_cls_int = OC_RP_XSF.value
```

Comment:
I'm surprised this didn't cause test errors.  Does every call to `DaosObj.create()` pass in an `objcls`? I would have thought one would need to either use `DoasObjClass.OC_RP_XSF` or `pydaos_shim.OC_RP_XSF` to get this enum value.

Reply:
Putting it back, likely being missed in the middle of different approaches of DaoObjClass handling. But don't have answer yet to why the test didn't catch it.